src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/LagPortSpec.groovy

```diff
@@ -586,6 +586,42 @@ class LagPortSpec extends HealthCheckSpecification {
         lagPort && northboundV2.deleteLagLogicalPort(sw.dpId, lagPort)
     }
 
+    @Tidy
+    def "Unable decrease bandwidth on LAG port lower than connected flows bandwidth sum"() {
+        given: "Flows on a LAG port with switch ports"
+        def switchPair = topologyHelper.getSwitchPairs().first()
+        def testPorts = topology.getAllowedPortsForSwitch(switchPair.src).takeRight(2).sort()
+        assert testPorts.size > 1
+        def payload = new LagPortRequest(portNumbers: testPorts)
+        def lagPort = northboundV2.createLagLogicalPort(switchPair.src.dpId, payload).logicalPortNumber
+        def flow = flowHelperV2.randomFlow(switchPair).tap {
+            source.portNumber = lagPort
+            it.maximumBandwidth = 10000001
```

Comment:
Not quite obvious where this number comes from. Is it possible that the test picks two 40G ports to create LAG on hardware switch? Test will fail in this case

Reply:
Do not see any possibility to get port bandwidth via northbound or grpc so the answer is yes - it will fail if we get 40G ports for test. Any suggestions on that?

Reply:
I will take a look on fl api and anyway try to avoid hardcode for bw

Reply:
I found how to get port bw so uploaded fix. Maybe closure can be written better, more like groovy way